client/src/main/java/org/apache/cassandra/sidecar/client/SidecarClient.java

```diff
@@ -567,6 +566,28 @@ public void streamCdcSegments(SidecarInstance sidecarInstance,
                 .build(), streamConsumer);
     }
 
+    /**
+     * Sends a request to trigger an immediate, synchronous schema
+     * conversion and report on the specified instance of the Sidecar
+     * regardless of the periodic task schedule or status
+     *
+     * @param instance the {@link SidecarInstance} to receive the request
+     * @return a {@link CompletableFuture} for the request
+     */
+    public CompletableFuture<Void> reportSchema(SidecarInstance instance)
+    {
+        // Create an instance of {@link RequestContext.Builder} using its
+        // constructor instead of the {@link this.requestBuilder()} method,
+        // since {@link NoRetryPolicy} is the preferred behavior here
+
+        RequestContext context = new RequestContext.Builder()
+                .singleInstanceSelectionPolicy(instance)
+                .reportSchemaRequest()
+                .build();
+
+        return executor.executeRequestAsync(context);
```

Comment:
you can always override the policy like this:
```suggestion
        return executor.executeRequestAsync(requestBuilder()
                                            .singleInstanceSelectionPolicy(instance)
                                            .retryPolicy(new NoRetryPolicy())
                                            .reportSchemaRequest()
                                            .build());
```

Reply:
Added a convenience method for policy overrides following the existing pattern.